### ISSUE TITLE
Top Member Role does not show when using MemberCount 

### ISSUE THREAD

**Issue by DevilHunter1337** [Arashiryuu/crap#35]
Not sure when this happen, but when using the plugin MemberCount, the Top Role of any member on discord will not show. Instead, the member just appears on its own.. I tried disabling every plugin, and only happens with MemberCount. Even deleted the file and redownloaded it. here is a few screenshots showing what I mean.. 

[Screenshot 1 - Top Role not showing](https://www.dropbox.com/s/z7jikd02w9d7bma/Screenshot_62.png?dl=0)

[Screenshot 2 - Showing my Role list for my server. I am the only one in The Overseer group, and thus top role](https://www.dropbox.com/s/znxml55u242r0uj/Screenshot_63.png?dl=0)

**Comment by Arashiryuu** [comment]
I have never had or been able to reproduce this, so tell me if this update changes it for you.

**Comment by DevilHunter1337** [comment]
Nope, still the same. Tried it with all plugins off, cept membercount. Even tried doing CTRL + R, and fully closing discord and reopening it.

**Comment by DevilHunter1337** [comment]
I tried changing the value to even 10 and even lower.. no effect. I have no clue what is causing this but.. it seems the last few updates broke it, either that or Discord changed things around. A few other Plugins that I use are also broken, thus refuse to start or just don't work right. I tried everything last night, even as far as Removing betterdiscord and discord itself off and reinstalling one by one.. no Dice.
